src/Component/src/Metadata/Resource/Factory/AttributesResourceMetadataCollectionFactory.php

```diff
@@ -143,20 +149,6 @@ private function getOperationWithDefaults(ResourceMetadata $resource, Operation
 
         $operation = $operation->withResource($resource);
 
-        if (null === $resource->getName()) {
-            $resourceName = $resourceConfiguration->getName();
-
-            $resource = $resource->withName($resourceName);
-            $operation = $operation->withResource($resource);
-        }
-
-        if (null === $resource->getPluralName()) {
```

Comment:
I guess plural name is set somehow else, so this code is removed, but I don't fully understand how or where.